scanomatic/ui_server_data/js/ccc/components/Plate.jsx

```diff
@@ -7,6 +7,9 @@ const SELECTED_COLONY_MARKER_COLOR = '#c82124';
 const SELECTED_COLONY_MARKER_STROKE_WIDTH = 5;
 const SCALE = 0.2;
 
+/* eslint no-param-reassign:
+  ["error", { "props": true, "ignorePropertyModificationsFor": ["context"] }]
+*/
```

Comment:
did you do an issue for this, if not please make one

Reply:
I have now: https://github.com/Scan-o-Matic/scanomatic-standalone/issues/66